### PR TITLE
Updated ingress applied

### PR DESCRIPTION
**Pull request by rohanssalunkhe**
## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [ ] (3) Tests passing
* [ ] (4) Branch ready to be merged (not work in progress)
* [ ] (5) No superfluous changes in diff
* [ ] (6) No TODO's without new ticket numbers
* [ ] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`
* [ ] (8) Data migration script is created if any of letter templates is changed

### CODE REVIEW

**Review by vertism — APPROVED** [review]

File: config/kubernetes/demo/ingress-live.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ metadata:
     external-dns.alpha.kubernetes.io/set-identifier: track-a-query-ingress-track-a-query-demo-green
     external-dns.alpha.kubernetes.io/aws-weight: "100"
```
Comment (digitalali-moj) [inline config/kubernetes/demo/ingress-live.yaml:10]: Question about this - should it be zero?

Comment (rohanssalunkhe) [reply config/kubernetes/demo/ingress-live.yaml:10]: no 100 is correct

File: config/kubernetes/development/ingress-live.yaml [hunk 2]
```diff
@@ -17,7 +18,11 @@ spec:
     - host: development.track-a-query.service.justice.gov.uk
       http:
         paths:
-          - path: /
+        - path: /
```
Comment (digitalali-moj) [inline config/kubernetes/development/ingress-live.yaml:21]: should this be indented and dash?

**Review by digitalali-moj — APPROVED** [review]
Looks good - just a couple of things I want you to double check please.

**Pull request merged**